regression/goto-analyzer/liveness/test-liveness.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--variable-sensitivity --vsd-values set-of-constants --vsd-liveness --show
+^EXIT=0$
+^SIGNAL=0$
+^main::1::x .* TOP @ \[1\]
+^main::1::x .* value-set-begin: 0 :value-set-end @ \[2\]
+^main::1::x .* value-set-begin: 2, 3 :value-set-end @ \[7\]
```

Comment:
Pedantry probably but could we have the other two regions for illustration and completeness.